en/docs/security-announcements/cve-justifications/2025/CVE-2021-42646.md

```diff
@@ -0,0 +1,53 @@
+---
+title: CVE-2021-42646
+category: security-announcements
+date: June 02, 2025
+---
+
+# CVE-2021-42646
+
+<p class="doc-info">WSO2 Products Impacted: No</p>
+<p class="doc-info">Customers Actions Required: No</p>
+---
+
+### REPORTED VULNERABILITY
+An XML External Entity (XXE) vulnerability in the file-based service provider creation feature of the WSO2 Management Console allows attackers to send crafted GET requests to access sensitive files or cause denial of service by exploiting insecure XML parsing [^1].
+
+### REPORTED PRODUCTS
+- WSO2 API Manager: 2.6.0, 3.0.0, 3.1.0, 3.2.0, 4.0.0  
+- WSO2 Identity Server: 5.7.0, 5.8.0, 5.9.0, 5.10.0, 5.11.0  
+- WSO2 Identity Server as Key Manager: 5.7.0, 5.9.0, 5.10.0
+- WSO2 Open Banking KM: 1.3.0, 1.4.0, 1.5.0
+- WSO2 Open Banking AM: 1.3.0, 1.4.0, 1.5.0, 2.0.0
+- WSO2 Open Banking IAM: 2.0.0
+
+### WSO2 JUSTIFICATION
+The vulnerability mentioned in CVE-2021-42646 has been addressed and fixed in the respective versions as per the advisory published at WSO2-2021-1289 [^2]. The affected components and their corresponding fixed versions are listed below:
+
+### Affected Components and Fixed Versions:
+
+#### WSO2 Carbon Identity Application Management (`org.wso2.carbon.identity.application.mgt`)
+
+Unaffected ranges are,
+
+| version (start range) | <= version (range) | Version Type |
+|:---------------------:|:------------------:|:------------:|
+| 5.12.153.10           | 5.12.153.*         | Commercial   |
+| 5.12.387.4            | 5.12.387.*         | Commercial   |
+| 5.14.97.8             | 5.14.97.*          | Commercial   |
+| 5.17.5.59             | 5.17.5.*           | Commercial   |
+| 5.18.187.53           | 5.18.187.*         | Commercial   |
+
+### CONCLUSION
+
+Based on the following key points:
+
+- The incorrect listing of `org.wso2.carbon.identity.framework:org.wso2.carbon.identity.application.mgt_5.17.118` as a vulnerable component version.
```

Comment:
This is not an incorrect listing, isn't it? according to the PR, `org.wso2.carbon.identity.framework:org.wso2.carbon.identity.application.mgt` is the vulnerable component we fixed. Please correct me if I'm wrong

Reply:
Instead of that, we can use the following I think. 

```
- The omission of commercial patch versions in external advisories.
```

Reply:
IMO, we can use the 2nd and 3rd points of the https://security.docs.wso2.com/en/latest/security-announcements/cve-justifications/2024/CVE-2022-29464/#conclusion here and may be as the third point. Just a suggestion. You can apply them based on your judgements

Reply:
Yes , your point is correct, here i have listed the version for the 3.2.1. I will change this